apps/web/src/lib/components/book-reader/book-reader-paginated/book-reader-paginated.svelte

```diff
@@ -432,6 +433,7 @@
 
   .book-content-container {
     column-count: var(--book-content-column-count, 1);
+    column-width: var(--book-content-child-width-single-column, auto); //necessary for the page to render correctly on ios
```

Comment:
Update this line to
`column-width: var(--book-content-child-column-width, auto); // required for WebKit + column-count 1`